modules/rules_appimage/1.7.4/source.json

```diff
@@ -0,0 +1,10 @@
+{
+    "integrity": "sha256-04vid8qIukiX0ekvqiSrFHzg30HaptWi2cZsSKpUcio=",
+    "strip_prefix": "rules_appimage-1.7.4",
+    "url": "https://github.com/lalten/rules_appimage/archive/refs/tags/v1.7.4.tar.gz",
```

Comment:
Could you add a release archive for this tag? The `archive` endpoint is really just a fallback solution as it doesn't necessarily have a stable hash.

Reply:
Done, need to look into somehow generating this automatically via CI in the future

Reply:
You may want to checkout how many rulesets do this, just by calling `git archive`.  See https://github.com/bazel-contrib/rules-template/blob/main/.github/workflows/release.yml
We should probably write some actual documentation for it as well.

Reply:
Thanks for the link! I'd suggest we land this PR as is and I'll work on the automation for the next release.